### PR TITLE
V0.1 LED Tophat Mod

### PR DESCRIPTION
**Pull request by Shteeeeeve**
* [ X] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [X ] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [X ] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [X ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ X] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ X] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [X ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [X ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [X ] Submitted STL files are oriented and scaled properly for printing.
  * [X] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

Design a tophat with integrated LEDs using 24V lightstrip (the one I am using is linked in the readme.md file.

#### Which mods/configurations/profiles are added by this PR?
User Mods

#### How was it tested? 
Printed / Tested on my V0.1

#### Any background context you want to provide?
NA

#### Screenshots (if appropriate)
https://discord.com/channels/460117602945990666/624317225540780055/861410849494794310

#### Further notes
NA

### CODE REVIEW

**Comment by Shteeeeeve** [comment]
First mod / pull request -- not sure if I'm required required to report a status?

File: printer_mods/Shteeeeeve/V01-LED-Tophat/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## Printed Parts:
+1. LED Tophat - Main Body
+2. LED Tophat - Top Cover
+3. LED Tophat - Logo Insert
+
+## Parts:
+1. 4x M3x6 BHCS
+2. 4x M3x5x4 Heat Inserts
+3. 24V LED Lightstrip \\\ URL: https://tinyurl.com/7axzaff4
```
Comment (KiloQubit) [inline printer_mods/Shteeeeeve/V01-LED-Tophat/README.md:13]: Please replace this with a plain Amazon link, looks like it should be

    https://www.amazon.ca/gp/product/B074XCKJYB

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Please remove spaces from all filenames.

It would be nice to include a picture of how it looks on the printer with LED on.

**Comment by Shteeeeeve** [comment]
@KiloQubit -- changed link, added photo, and removed spaces from STL and STEP files. Changes committed. Thanks!

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!

**Pull request merged**